modules/proc_configuring-notificationsconfiguration-crd-by-using-the-ocp-cli.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * argocd_instance/argo-cd-cr-component-properties.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-notificationsconfiguration-cr-by-using-cli_{context}"]
+= Configuring the NotificationsConfiguration CR by using CLI
+
+You can configure the `NotificationsConfiguration` CR by using CLI.
+
+.Prerequisites
+
+* You have access to an {OCP} cluster with `cluster-admin` privileges.
+* You have installed the {gitops-title} Operator on your cluster.
+* You have enabled notifications for the Argo CD instance.
+
+.Procedure
+
+. Enable notifications for the Argo CD instance in the CLI:
++
+[source,terminal]
+----
+$ oc edit argocd openshift-gitops -n openshift-gitops
+----
++
+where:
+
+`argocd openshift-gitops`:: The name of the Argo CD instance.
+`openshift-gitops`:: The name of the namespace.
+
++
+.Example output
+[source,terminal]
+----
+notifications:
+  enabled: true
+----
+
+. Edit the default `NotificationsConfiguration` CR in the cluster:
++
+[source,terminal]
+----
+$ oc edit notificationsconfiguration default-notifications-configuration -n openshift-gitops
+----
++
+where:
+
+`default-notifications-configuration`:: The name of the default `NotificationsConfiguration` CR.
+`openshift-gitops`:: The name of the namespace.
```

Comment:
Same comment as above about using user-replaced values, if applicable:
```suggestion
$ oc edit notificationsconfiguration <default_notifications_configuration_CR> -n <namespace>
----
+
where:

`<default_notifications_configuration_CR>`:: The name of the default `NotificationsConfiguration` CR.
`<namespace>`:: The name of the namespace.
```

Reply:
I cannot edit `default-notifications-configuration` to what you are suggesting @skopacz1 but I can change  "openshift-gitops" to "namespace".

Reply:
Fixed.